lib/transform/allOfTransformer.ts

```diff
@@ -18,6 +18,9 @@ const transformAllOfSchema = (schema: Schema, baseSchemas: Set<Schema>, jsonLoad
     transformAllOfSchema(sch, baseSchemas, jsonLoader);
 
     const { properties, required, additionalProperties: aProperties } = sch;
+    if (sch["x-ms-azure-resource"] === true) {
+      schema["x-ms-azure-resource"] = true;
```

Comment:
What if schema["x-ms-azure-resoutce"] is false? Do we intend to overwrite it?

Reply:
Consider use the variable xmsAzureResource to replace the string

Reply:
Override it only when undefined.